frontend/packages/noobaa-storage-plugin/integration-tests/views/createBC.view.ts

```diff
@@ -0,0 +1,159 @@
+  const defaultBS = JSON.parse(
+    execSync(
+      'kubectl get backingstores.noobaa.io noobaa-default-backing-store -n openshift-storage -o json',
+    ).toString(),
+  );
+  const customObj = _.pick(defaultBS, ['apiVersion', 'kind', 'metadata', 'spec']);
+  customObj.metadata = Object.assign(_.pick(customObj.metadata, ['name', 'namespace']), {
+    name,
+  });
+  execSync(`echo '${JSON.stringify(customObj)}' | kubectl create -f -`);
+};
+
+const tierLevelToButton = (level: number, tier: Tier) =>
+  level === 1
+    ? tier === Tier.SPREAD
+      ? tier1Spread
+      : tier1Mirror
+    : tier === Tier.SPREAD
+    ? tier2Spread
+    : tier2Mirror;
+
+export class BucketClassHandler {
+  name: string;
+
+  namespace: string;
+
+  description: string;
+
+  tiers: Tier[];
+
+  constructor(name: string, description: string, namespace: string = 'openshift-storage') {
+    this.name = name;
+    this.namespace = namespace;
+    this.description = description;
+    // Create 3 backingStores similar to default backing store
+    for (let count = 0; count < 3; count++) {
+      createDummyBackingStore(`${testName}-${count}`);
+    }
+  }
+
+  setTiers(tiers: Tier[]) {
+    this.tiers = tiers;
+  }
+
+  async createBC() {
+    // Select namespace
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    // General Data Page
+    await this.setGeneralData();
+    await click(nextButton);
+    // Placement Policy Page
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await this.setPlacementPolicy();
+    await click(nextButton);
+    // Backing Store Selection Page
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await this.setBackingStores();
+    await click(nextButton);
+    // Review Page
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    const data = await this.getReviewPageData();
+    await click(createBucketClassButton);
+    return data;
+  }
+
+  async setGeneralData() {
+    await bucketClassNameInput.sendKeys(this.name);
+    await bucketClassDescriptionInput.sendKeys(this.name);
+  }
+
+  async setPlacementPolicy() {
+    await click(tierLevelToButton(1, this.tiers[0]));
+    if (this.tiers.length > 1) {
+      await click(addTierButton);
+      await click(tierLevelToButton(2, this.tiers[1]));
+    }
+  }
+
+  async setBackingStores() {
+    // Wait until the table gets populated
+    await browser.wait(
+      until.and(
+        async () =>
+          getBackingStoreRequiredCount(this.tiers) <= (await tier1TableCheckBoxes.count()),
+      ),
+    );
+    // Select tier 1 Backing Stores
+    await click(tier1TableCheckBoxes.get(0));
+    if (TierCountMap[this.tiers[0]] > 1) await click(tier1TableCheckBoxes.get(1));
+    // Select tier 2 Backing Stores
+    if (this.tiers.length > 1) {
+      await click(tier2TableCheckBoxes.get(0));
+      if (TierCountMap[this.tiers[1]] > 1) await click(tier2TableCheckBoxes.get(1));
+    }
```

Comment:
It would be good if we can also check for working of create backing store link.

Reply:
click does that.